src/command-configs/renderHelpPage.pug

```diff
@@ -0,0 +1,168 @@
+html
+  head
+    title Help - Command Bot
```

Comment:
Something could be odd / better. This is temp improvement before making a UI version, which won't require "documentation"